src/Umbraco.Infrastructure/RuntimeOptions.cs

```diff
@@ -1,74 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using Umbraco.Core.Cache;
-using Umbraco.Core.Composing;
-using Umbraco.Core.Logging;
-using Umbraco.Core.Persistence;
-
-namespace Umbraco.Core
-{
-    /// <summary>
-    /// Provides static options for the runtime.
-    /// </summary>
-    /// <remarks>
-    /// These options can be configured in PreApplicationStart or via appSettings.
-    /// </remarks>
-    public static class RuntimeOptions
```

Comment:
So all this stuff is a hack and mainly for Umbraco Deploy as it was in v8 since it needs to very early on replace services before the light inject container is built, etc...  We'll have to revisit Deploy, etc... much later and see how these old hacks will translate to this new version but "essentially" Deploy could configure a connection string dynamically and ensure a DB exists before calls to run the RuntimeState checks. Just wanted to mention this here for info. Totally fine we remove this all for now and figure out what Deploy should do later.

Reply:
Yeah you shouldn't need hacks anymore!!